EditorExtensions/Shared/Compilers/ServerBase.cs

```diff
@@ -92,8 +91,11 @@ private void Dispose(bool disposing)
             {
                 using (await mutex.LockAsync())
                 {
-                    if (server == null || server._process == null || server._process.HasExited)
+                    if (server == null || server._address == null || server._process == null || server._process.HasExited)
+                    {
                         server = new T();
+                        server._address = string.Format(CultureInfo.InvariantCulture, "http://localhost.:{0}/", server.BasePort);
```

Comment:
I have actually a case where the localhost with point is going into a proxy request. Beside that, I don't get why the point is there anyway. Can you elaborate?